sources/platform/console/settings.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Settings
+description: Learn about the Apify Console settings for the account, integrations, organization management, and notification preferences.
+sidebar_position: 3
+category: platform
+slug: /console/settings
+---
+
+**Learn about the Apify Console settings for the account, integrations, organization management, and notification preferences.**
+
+---
+
+## Account
+
+By clicking the **Settings** tab on the side menu, you will be presented with an Account page where you can view & edit various settings regarding your account, such as:
+
+* account email
+* username
+* profile information
+* theme
+* login information
+* account delete
+
+## Integrations
+
+The **Integrations** tab provides essential tools for enhancing your interaction with our platform. Here, you can access your **Personal API Tokens**, which are necessary for using our [REST API](https://docs.apify.com/api/v2). This page also facilitates the integration of your Slack workspace and lists your **Actor Integration Accounts**. This section represents any third-party integrations added by you or your team. For detailed guidance on utilizing these integrations, refer to our [Integrations documentation](https://docs.apify.com/platform/integrations).
+
+## Organization
+
+The **Organizations** tab is where you can view your accounts' current organizations, create new organizations, or convert your user account into an organization account. For more information on organizations, visit the organization's [documentation](https://docs.apify.com/platform/collaboration/organization-account).
```

Comment:
I was told this documentation is outdated (at least the screenshots). This help article https://help.apify.com/en/articles/8698948-how-to-set-up-an-organization-account was written recently, so you could link to that instead of the docs. However, whether you link to the docs or the help article, the docs should be updated if they're out of date.

Reply:
Thank you I will link to the help article for now and try to give this doc a higher priority for refresh